storefront/spec/controllers/spree/orders_controller_spec.rb

```
@@ -120,6 +120,18 @@
 
+
+    context 'when order does not require ship address' do
+      let(:digital_shipping_method) { create(:digital_shipping_method) }
+      let(:digital_product) { create(:product, shipping_category: digital_shipping_method.shipping_categories.first) }
+      let(:order) { create(:completed_order_with_totals, store: store, user: user, line_items: [create(:line_item, variant: digital_product.master)]) }
+
+      it 'renders the show template' do
+        get :show, params: { id: order.number, token: order.token }
+        expect(response).to render_template(:show)
+      end
+    end
```

RuboCop (1.75.5)

[convention] Example group has too many memoized helpers [7/5]

(RSpec/MultipleMemoizedHelpers)